homeassistant/components/sensor/rmvtransport.py

```diff
@@ -32,7 +32,18 @@
 
 DEFAULT_NAME = 'RMV Journey'
 
-DEFAULT_PRODUCT = ['U-Bahn', 'Tram', 'Bus', 'S', 'RB', 'RE', 'EC', 'IC', 'ICE']
+VALID_PRODUCTS = {
```

Comment:
I don't think that is right.

I used following code to test
```
>>> import voluptuous as vol
>>> import homeassistant.helpers.config_validation as cv
>>> LIST = ['a','b','c']
>>> schema = vol.Schema(vol.All(cv.ensure_list, [vol.In(LIST)]))
>>> schema('a')
['a']
>>> schema(['a'])
['a']
>>> schema(['a','b'])
['a', 'b']
>>> schema(['a','b','d'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jason/ha/home-assistant/venv/lib/python3.6/site-packages/voluptuous/schema_builder.py", line 267, in __call__
    return self._compiled([], data)
  File "/home/jason/ha/home-assistant/venv/lib/python3.6/site-packages/voluptuous/validators.py", line 204, in _run
    return self._exec(self._compiled, value, path)
  File "/home/jason/ha/home-assistant/venv/lib/python3.6/site-packages/voluptuous/validators.py", line 286, in _exec
    raise e if self.msg is None else AllInvalid(self.msg, path=path)
  File "/home/jason/ha/home-assistant/venv/lib/python3.6/site-packages/voluptuous/validators.py", line 284, in _exec
    v = func(path, v)
  File "/home/jason/ha/home-assistant/venv/lib/python3.6/site-packages/voluptuous/schema_builder.py", line 641, in validate_sequence
    raise er.MultipleInvalid(errors)
voluptuous.error.MultipleInvalid: value is not allowed @ data[2]

```

If config validation script has issue, you may need look at that script

Reply:
It must be a dictionary not a list. As far as I can see it is working as expected.